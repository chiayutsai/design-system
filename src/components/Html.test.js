import renderer from 'react-test-renderer'
import Html from './Html'

describe('Html', () => {
  test('render correctly', () => {
    const wrapper = renderer
      .create(<Html title="title" description="description" />)
      .toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})
