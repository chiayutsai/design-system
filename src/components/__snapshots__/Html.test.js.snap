// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Html render correctly 1`] = `
<html
  className="no-js"
  lang="en"
>
  <head>
    <meta
      charSet="utf-8"
    />
    <meta
      content="ie=edge"
      httpEquiv="x-ua-compatible"
    />
    <title>
      title
    </title>
    <meta
      content="description"
      name="description"
    />
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <script
      dangerouslySetInnerHTML={
        Object {
          "__html": "
              window.version = \\"2.3.0\\";
              var querycdn = new URL(window.location).searchParams.get(\\"cdnurl\\") || \\"\\";
              var publicpath = \\"./assets/\\";
              window.__CDN_URL_PATH__ = querycdn.replace(/\\\\/$/, \\"\\") + publicpath.replace(querycdn ? \\".\\" : \\"\\", \\"\\");
              ",
        }
      }
    />
    <link
      href="/icon.png"
      rel="apple-touch-icon"
    />
  </head>
  <body>
    <div
      id="app"
    />
  </body>
</html>
`;
